liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/AbstractLibertySupport.java

```diff
@@ -63,6 +64,9 @@
     @Parameter(defaultValue = "${session}", readonly = true)
     protected MavenSession session;
     
+    @Parameter(property = "reactorProjects", required = true, readonly = true)
+    protected List<MavenProject> reactorProjects;
```

Comment:
What does it mean to define reactor projects? Why do they require configuration? I don't see tests to help understand what these projects are about.

Reply:
Reactor projects are the projects that will be built during maven build session.  For a single project build, it contains only the current project.  For a multi-module projects build (i.e. mvn against the top level project), the reactor projects contains top level project and all its sub module projects.  

Maven runtime will inject the value of `MavenSession.getProjects()` into `reatorProjects` object.  see https://maven.apache.org/ref/3.5.0/maven-core/apidocs/org/apache/maven/plugin/PluginParameterExpressionEvaluator.html

Reply:
Ah - OK, so this is not something they are required to configure?